package.json

```diff
@@ -1,148 +1,148 @@
-    "lodash": "^4.17.14",
-    "wordpress-element": "npm:@wordpress/element@2.11.0"
-  },
-  "devDependencies": {
-    "@automattic/color-studio": "2.3.1",
-    "@automattic/puppeteer-utils": "github:Automattic/puppeteer-utils",
-    "@babel/plugin-proposal-object-rest-spread": "7.12.13",
-    "@testing-library/jest-dom": "5.14.1",
-    "@testing-library/react": "11.2.5",
-    "@testing-library/user-event": "10.4.1",
-    "@types/lodash": "^4.14.170",
-    "@types/react": "16.14.5",
-    "@types/wordpress__components": "^9.8.6",
-    "@types/wordpress__date": "^3.3.2",
-    "@typescript-eslint/eslint-plugin": "4.15.2",
-    "@typescript-eslint/parser": "4.15.2",
-    "@woocommerce/api": "^0.2.0",
-    "@woocommerce/components": "5.1.2",
-    "@woocommerce/currency": "3.0.0",
-    "@woocommerce/date": "2.1.0",
-    "@woocommerce/e2e-utils": "^0.1.5",
-    "@woocommerce/eslint-plugin": "1.1.0",
-    "@woocommerce/experimental": "1.3.0",
-    "@wordpress/api-fetch": "5.1.1",
-    "@wordpress/babel-plugin-makepot": "2.1.3",
-    "@wordpress/babel-preset-default": "4.3.0",
-    "@wordpress/base-styles": "^3.5.4",
-    "@wordpress/browserslist-config": "3.0.1",
-    "@wordpress/components": "11.1.5",
-    "@wordpress/data": "4.22.1",
-    "@wordpress/data-controls": "1.6.0",
-    "@wordpress/date": "3.9.0",
-    "@wordpress/dependency-extraction-webpack-plugin": "3.0.1",
-    "@wordpress/e2e-test-utils": "4.11.0",
-    "@wordpress/element": "^2.20.3",
-    "@wordpress/i18n": "^4.1.1",
-    "@wordpress/icons": "^2.10.3",
-    "@wordpress/jest-preset-default": "4.0.0",
-    "@wordpress/scripts": "12.4.0",
-    "archiver": "5.2.0",
-    "babel-loader": "8.2.2",
-    "classnames": "^2.3.1",
-    "colors": "1.4.0",
-    "config": "3.3.6",
-    "core-js": "3.9.0",
-    "dotenv": "8.2.0",
-    "eslint": "7.30.0",
-    "eslint-config-prettier": "7.2.0",
-    "eslint-config-wpcalypso": "4.0.1",
-    "eslint-import-resolver-typescript": "^2.4.0",
-    "eslint-loader": "4.0.2",
-    "eslint-plugin-jest": "24.3.6",
-    "eslint-plugin-jsdoc-alignment": "0.0.5",
-    "eslint-plugin-jsx-a11y": "6.4.1",
-    "eslint-plugin-react": "7.22.0",
-    "eslint-plugin-react-hooks": "4.2.0",
-    "eslint-plugin-wpcalypso": "5.0.0",
-    "husky": "6.0.0",
-    "is-ci": "^3.0.0",
-    "lint-staged": "10.5.4",
-    "mini-css-extract-plugin": "1.3.8",
-    "node-sass": "5.0.0",
-    "node-wp-i18n": "^1.2.5",
-    "postcss": "^8.3.5",
-    "prettier": "npm:wp-prettier@2.0.5",
-    "react": "16.14.0",
-    "react-dom": "16.14.0",
-    "react-test-renderer": "16.14.0",
-    "sass-loader": "10.1.1",
-    "shelljs": "0.8.4",
-    "source-map-loader": "^1.1.3",
-    "stylelint": "13.11.0",
-    "stylelint-config-prettier": "8.0.2",
-    "stylelint-config-wordpress": "17.0.0",
-    "ts-jest": "26.5.6",
-    "ts-loader": "^8.3.0",
-    "typescript": "4.3.5",
-    "webpack": "^4.46.0",
-    "webpack-cli": "4.5.0",
-    "yarnhook": "^0.5.1"
-  }
+	"name": "woocommerce-payments",
```

Comment:
I see mixed usage of tabs/spaces. Specially on .json files. Which one is correct?

--

Apparently, [tabs is correct](https://make.wordpress.org/core/handbook/best-practices/coding-standards/javascript/#spacing ). Is it OK to update the whole file though?

Reply:
@reykjalin: can we keep the changes in `package.json` minimal, and keep the original spacing for the sake of minimalistic diff?

Reply:
Ugh, sorry about that. My editor is set up to run prettier on JSON files too; I forgot about it here.

I'll fix this and merge 👍